vocabularies/Common.json

```
@@ -1546,6 +1546,18 @@
             "@Common.Experimental": true,
             "@Core.Description": "Sorting and filtering of amounts in multiple currencies needs special consideration",
             "@Core.LongDescription": "TODO: add link to UX documentation on https://experience.sap.com/fiori-design/"
+        },
+        "mediaUploadURL": {
```

```suggestion
        "MediaUploadURL": {
```
Most other terms in this vocabulary are capitalized, so I'd suggest doing this here as well.

---

All newer instance annotations in this vocabulary are lowerCamelCase.

---

Maybe we should actually call it `mediaUploadLink` to be in sync with the `mediaReadLink` and `mediaEditLink`.

---

Ok, I did not know about the naming convention for instance annotations.
Yes, mediaUploadLink might make sense then.

---

Could a mediaUploadLink also point to an HTML page where users must use an HTML form with a file uploader to submit their new file? That would be clear. Or is the expectation that the file is uploaded via a REST call?